server/graphql/v2/object/Order.js

```diff
@@ -149,6 +150,13 @@ export const Order = new GraphQLObjectType({
           }
         },
       },
+      hostFeePercent: {
+        type: GraphQLInt,
+        description: 'Host fee percent attached to the Order.',
+        async resolve(order) {
+          return await getHostFeePercent(order);
+        },
+      },
```

Comment:
Maybe a good opportunity to introduce https://github.com/opencollective/opencollective/issues/5606?